### DISCUSSION TITLE
Feature request for PC player

### DISCUSSION THREAD

**Discussion by watanyoshi** [GerryDazoo/SlingerPlayer#5] [Ideas]
I started using SlingerPlayer and love it so much. It has very simple UI but meets almost all my operation needs. I just want to add a couple of improvement ideas.
1. To exit full screen mode, can you add double click as addition to ESC key input? Pressing ESC is a bit pain when watching video far from keyboard.
2. Can you add the feature to remember Remote control mode On or Off in non full screen mode. Currently Remote is always off after exiting Full screen mode. Exiting full screen mode is for using remote control in most cases with me. It may improve usability for someone like me to watch recorded programs with commercials to be skipped.

**Comment by watanyoshi** [comment]
One more feature nice to have is ability to set URLbase for Remote controller. Currently it is fixed as "Remote". To make it safer from interuption, Slinger user may want to change it to something different.

**Comment by GerryDazoo** [comment]
Requested Remote changes in latest version. I also added the ability to start the remote in Full Screen mode with a single click or press "r" key.

**Reply by watanyoshi** [reply]
Hi Gerry, thanks a lot for adding the remote changes. AoT + double click works perfectly for my usage. I hope this version of SlingerPlayer helps other people who are not satisfied with VLC or other players. Customizing own remote controller is fun and I think I like SlingerPlayer better than the orginal Sling Player.

**Comment by as400jockey** [comment]
I love this gadget. TV in one window, remote in another.  Beautiful!

The only thing I would want would be the ability to Cast to Chromecast.  I currently have 5 cable boxes at $10/ea/mo and only myself and my wife here.  Slingbox and the firestick app (or was it the Roku app) was supposed to allow me to get rid of a box or two.  Until Sling killed the app on one of those two.  Then I bought the other, and they killed that one too.  Then they killed the android app and finally the entire thing.

So this works beautifully for watching from my desktop or laptop.

I found my old FireStick and have your apk running on that right now!  Another triumph.  So that'll take care of one TV.  Will run the remote from my phone.
